src/pages/AppDetails/AppInfo/AppDescription.tsx

```diff
@@ -132,13 +136,23 @@ class AppDescription extends React.Component<AppDescriptionProps, AppDescription
           </Card>
         </GridItem>
         <GridItem span={6}>
-          <Card>
+          <Card style={{ height: '100%' }}>
             <CardBody>
-              <div>
-                <Text component={TextVariants.h2}>Health Overview</Text>
-                <strong>Health</strong>
-              </div>
-              <HealthIndicator id={app.name} health={this.props.health} mode={DisplayMode.LARGE} />
+              <Title headingLevel="h3" size="2xl">
```

Comment:
Perhaps it could be a good moment to standardize also the `Application Overview` title in L118/122. I see that in [Workload overview](https://github.com/kiali/kiali-ui/blob/master/src/pages/WorkloadDetails/WorkloadInfo/WorkloadDescription.tsx#L55) is also following the html you just added here.